packages/plugins/i18n/server/src/register.ts

```diff
@@ -30,8 +30,17 @@ const isUrlForCreation = (url: string) => {
   // [ 'content-manager', 'collection-types', 'api::category.category' ]
   const splitUrl = url.split('/').filter(Boolean);
 
+  // Checking for query params and removed in the
+  const regex: RegExp = /[?]\S/;
+  const indexQueryParam = splitUrl.findIndex((item) => regex.test(item));
+
+  if (indexQueryParam !== -1) {
+    splitUrl.splice(indexQueryParam, 1);
+  }
+
```

Comment:
looks like this breaks in some scenarios:
`isUrlForCreation('/content-manager/collection-types/api::category.category?locale=en')`


what if we did something like this:
```ts
const isUrlForCreation = (url: string) => {
  if (!url) return false;

  // Remove any query params
  // /content-manager/collection-types/api::category.category/?locale=en
  const path = url.split('?')[0];

  // Split path and remove empty strings
  // [ 'content-manager', 'collection-types', 'api::category.category' ]
  const splitUrl = path.split('/').filter(Boolean);

  // Get the last element of the array
  // api::category.category / 1 / publish

  const model = splitUrl[splitUrl.length - 1];

  // If the model contains :: it means it's a uid
  return model.includes('::');
};
```

Reply:
I saw this message resolved but not any commit with a change, do you have any uncommited work?

I would like to include the fix in this week release!

Reply:
Thanks for letting me know. I've just pushed the change.